src/components/panels/NewsPanel.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+  <b-col md="6">
+        <h3 class="text-center">{{headertext}}</h3>
+        <b-carousel id="infoscreen-carousel" :interval="9000" :indicators="false" :controls="true">
+          <b-carousel-slide   v-for="(newsitem,index) in news" :key="index"
+          v-if="newsitem.translations['en']"
+          :text="newsitem.translations['en']">
+          <img slot="img" class="d-block   w-100" width="1024" height="250"
+          :src="newsitem.imageLink" alt="image slot">
+          <p style="color: 'black'!important">
```

Comment:
please do a dark shade with light text to follow the existing pattern for text on the site.

Reply:
alternatively, give both a class so we can change them depending on theme.